unfetter-discover/unfetter-ui
# Tightened CSS in sidebar for baselines and assessments v3

Fixes unfetter-discover/unfetter#1302

Thanks @mpmmasterpeace for the suggestion on this one, as it led to the overall solution.  Reduced the bottom padding of the expansion panel body, as well as the padding of the each `li` within the panel body.

### To test:
1. Pull this branch
2. Refresh the page if Unfetter already running
3. In Assessments beta and Baselines, edit an existing assessment/baseline
4. Notice in the sidebar the height of top level items as well as child items of the top level - both should essentially have the same height.
